tests/wasm/futures.rs

```diff
@@ -123,7 +123,6 @@ pub struct AsyncStruct;
 
 #[wasm_bindgen]
 impl AsyncStruct {
-    #[wasm_bindgen(constructor)]
```

Comment:
We need to keep this test until we actually remove the feature.
```suggestion
    #[wasm_bindgen(constructor)]
    #[allow(deprecated)]
```